Eatery Blue/UI/HomeScreen/HomeViewController.swift

```diff
@@ -19,9 +19,38 @@ class HomeViewController: UIViewController {
         case carouselView(CarouselView)
         case loadingView(CarouselView)
         case titleLabel(title: String)
+        case statusLabel(status: Status)
         case loadingLabel(title: String)
         case eateryCard(eatery: Eatery)
         case loadingCard
+        
+        func getEateryId() -> Int64?{
+            switch self {
+            case .eateryCard(let eatery):
+                return eatery.id
+            case .searchBar:
+                break
+            case .customView(view: _):
+                break
+            case .carouselView(_):
+                break
+            case .loadingView(_):
+                break
+            case .titleLabel(title: _):
+                break
+            case .statusLabel(status: _):
+                break
+            case .loadingLabel(title: _):
+                break
+            case .loadingCard:
+                break
+            }
+            return nil
+        }
+    }
+    enum Status {
+        case open
+        case closed
     }
```

Comment:
If this enum is not used anywhere else, mark it as `private`. This should also go at the top.

Reply:
Since cell uses Status making Status private means I would have to make Cell private as well which isn't possible

Reply:
Sounds good, just make sure to drag the enum to the top of the class.